doc/source/whatsnew/v0.17.0.txt

```diff
@@ -205,6 +205,52 @@ The support math functions are `sin`, `cos`, `exp`, `log`, `expm1`, `log1p`,
 These functions map to the intrinsics for the NumExpr engine.  For Python
 engine, they are mapped to NumPy calls.
 
+Changes to Excel with ``MultiIndex``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In version 0.16.2 a ``DataFrame`` with ``MultiIndex`` columns could not be written to Excel via ``to_excel``.
+That functionality has been added (:issue:`10564`), along with updating  ``read_excel`` so that the data can
+be read back with no loss of information by specifying which columns/rows make up the ``MultiIndex``
+in the `header` and `index_col` parameters (:issue:`4679`)
+
+.. ipython:: python
+
+    df = pd.DataFrame([[1,2,3,4], [5,6,7,8]],
```

Comment:
these should start right under the ``i`` of ipython (otherwise they indent too much)